src/views/Chat.js

```diff
@@ -60,6 +60,8 @@ const StyledChat = styled.div`
   display: flex;
   justify-content: space-around;
   flex-direction: row;
+  background-color: #;
```

Comment:
I see some of these are incomplete. Maybe comment them out until you're ready to use them as they might stop you from seeing other changes you're making.

Reply:
resolved